main.tex

```diff
@@ -55,32 +55,31 @@ \section{Introduction}
 Python 3 and is still maintained.
 
 
-\pyspeckit development began sometime in 2009 with a script called `showspec'
-in the \texttt{agpy} package (which was hosted on google code); i.e., it
-started as one graduate student's project to make a more general tool for
-plotting, and sometimes fitting profiles to, spectra in python.  At the time,
+\pyspeckit development began in 2009 with a script called `showspec'
+in the \texttt{agpy} package hosted on Google Code. It was created and used by
+a graduate student to plot and sometimes fit profiles to spectra in python. At the time,
 IDL was still more popular than python at most institutes \citep[first evidence
 that python had overtaken IDL in popularity among astronomers was presented in
 ]{Momcheva2015a}, and there were no publicly available and advertised tools for
 spectral plotting, fitting, and general manipulation \cite[\texttt{astropysics}
 was developed contemporaneously and solved many of the same problems as
-\pyspeckit][]{Tollerud2012a}.  The \astropy project had its first commit in
+\pyspeckit][]{Tollerud2012a}. The \astropy project had its first commit in
 2011, so even the basic infrastructure for such analysis was not yet
 established.
 
-\pyspeckit's GUI features were inspired by IRAF's \texttt{splot} tool, while
-the fitting features were inspired in part by \texttt{xspec}
+\pyspeckit's graphical user interface (GUI) features were inspired by IRAF's 
+\texttt{splot} tool, while the fitting features were inspired in part by \texttt{xspec}
 (\url{https://heasarc.gsfc.nasa.gov/xanadu/xspec/}).  Over subsequent years,
-\pyspeckit grew by incorporating more sophisticated models and improving its
+\pyspeckit grew by incorporating new models such as xx and improving its
```

Comment:
make the `xx` here `{\color{red} XX INSERT MODELS HERE XX}` or something that I won't miss